tests/internal/import_hooks/test_integration.py

```diff
@@ -0,0 +1,49 @@
+import contextlib
+import mock
+import sys
+
+import pytest
+
+from ddtrace.internal import import_hooks
+
+
+@pytest.fixture
+def hooks():
+    import_hooks.hooks.reset()
+    try:
+        yield import_hooks.hooks
```

Comment:
same as above

Reply:
For this one I want to explicitly test the global registry, unless you think that isn't necessary?